lib/outline.vim

```diff
@@ -151,7 +152,7 @@ def Open(): number
     # Set title
     setbufline(winbufnr(outline_win_id), 1, title)
     # Title does not follow syntax highlight but it is in black.
-    win_execute(outline_win_id, 'matchaddpos(''Terminal'', range(1,
+    win_execute(outline_win_id, 'matchaddpos(''Question'', range(1,
```

Comment:
I check the render with Question and it looks nice with both dark and light backgrounds (I am using everforest colorscheme). Nice!